Remotion/Web/UnitTests/Web.UnitTests.csproj

```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <Compile Include="..\..\..\SharedSource\Development\NUnit\UnitTesting\**\*.cs" Link="SharedSource\NUnit\UnitTesting\%(RecursiveDir)%(Filename)%(Extension)" />
     <Content Include="Res\*.xml" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="UrlMapping.xml" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
We wanted to have the default "UrlMapping.xml" as the default value for the urlmappingfile. We either need to move UrlMapping.xml to where it would be looked for in various tests or have some other default value for the test infrastructure.

Reply:
The existing tests used already an override to have it loaded form the Res-subfolder. I think it would be okay to keep the Res-subfolder and do a custom registration via ServiceLocatorScope or simply by injecting the custom settings object where the Res-subfolder is needed / causing an issue. Maybe show me the failing tests before investing time, though?

Reply:
Started a build with the urlmapping in res folder on teamcity